resistance/resitance.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program calculates the resistance R of two resistors.")
+	fmt.Println("Enter the resistance of two resistors:")
+	var a, b float64
+	fmt.Scanln(&a, &b)
+	R := a + b
+	R2 := 1/a + 1/b
```

Comment:
nit: This isn't resistance. It's called conductance! Please try to use appropriate letters for that.
https://en.wikipedia.org/wiki/Electrical_resistance_and_conductance